lib/broccoli/ember-app.js

```diff
@@ -1845,6 +1845,30 @@ function addOutputFile(container, assetPath, options) {
     container[outputFile] = [];
   }
 
+  if (~container[outputFile].indexOf(assetPath)) {
```

Comment:
I'd prefer a !== -1 here instead (until we can use includes). The bitwise operations always give me trouble when reading.